cnf-certification-test/platform/suite.go

```diff
@@ -173,3 +179,27 @@ func testTainted(env *provider.TestEnvironment) {
 	gomega.Expect(taintedNodes).To(gomega.BeNil())
 	gomega.Expect(errNodes).To(gomega.BeNil())
 }
+
+func testIsRedHatRelease(env *provider.TestEnvironment) {
+	ginkgo.By("should report a proper Red Hat version")
+	failedContainers := []string{}
+	for _, cut := range env.Containers {
+		ginkgo.By(fmt.Sprintf("%s(%s) is checked for Red Hat version", cut.Podname, cut.Data.Name))
```

Comment:
Please, add the namespace to this "By" statement (or use the cut.StringShort()). Also separate podname and cut name formatting chars with an extra space char: "%s (%s)"

Reply:
Added the `StringShort()` recommendation.  👍